iota-client/client.rs

```diff
@@ -72,9 +72,9 @@ impl Client {
     /// * [`uris`] - Slices of neighbor URIs(`&str`) to add
     ///
     /// [`uris`]: ../core/struct.AddNeighborsBuilder.html#method.uris
-    pub async fn add_neighbors(uris: &[&str]) -> Result<AddNeighborsResponse> {
-        for uri in uris {
-            match Url::parse(uri)?.scheme() {
+    pub async fn add_neighbors(uris: Vec<String>) -> Result<AddNeighborsResponse> {
```

Comment:
Is it has to be Vector? I was trying to avoid allocation as possible.

Reply:
I tried to convert a JsValue to a `&[&str]` but couldn't do it, maybe you could give it a try, I'm out of ideas there.

Reply:
Hmm, you are right. I was thinking `Box<[JsValue]>`, but it will do the allocation no matter what.